test/integration/super-sandbox-test.postman_collection.json

```diff
@@ -248,22 +248,22 @@
 					"listen": "test",
 					"script": {
 						"type": "text/javascript",
-						"exec": "console.log(request.headers);\nvar jsonData = JSON.parse(responseBody);\ntests[\"Correct aath header\"] = jsonData.headers.Authorization.indexOf(\"YWJoaWppdDprYW5l\")>-1;"
+						"exec": "console.log(request.headers);\nvar jsonData = JSON.parse(responseBody);\ntests[\"Authenticated\"] = jsonData.authenticated === true;"
 					}
 				}
 			],
 			"request": {
 				"auth": {
 					"type": "basic",
 					"basic": {
-						"username": "abhijit",
-						"password": "kane",
+						"username": "postman",
+						"password": "password",
 						"saveHelperData": true,
 						"showPassword": false
 					}
 				},
-				"url": "httpbin.org/post",
-				"method": "POST",
+				"url": "https://echo.getpostman.com/basic-auth",
```

Comment:
I think this tests for auth header being sent with POST requests. This change removes that aspect of the test.